main/lsp/lsp.cc

```diff
@@ -37,33 +38,32 @@ LSPQueryResult LSPLoop::queryByLoc(LSPTypechecker &typechecker, string_view uri,
     return typechecker.query(core::lsp::Query::createLocQuery(loc), {fref});
 }
 
-LSPQueryResult LSPLoop::queryBySymbol(LSPTypechecker &typechecker, core::SymbolRef sym,
-                                      optional<string_view> overSingleFile) const {
+LSPQueryResult LSPLoop::queryBySymbolInFiles(LSPTypecheckerDelegate &typechecker, core::SymbolRef sym,
```

Comment:
(I split this method into two separate methods since they didn't share much logic anyway. I can delurk if it's distracting from the main PR review.)

Reply:
Nah, this is fine...makes sense to me.